x-pack/plugins/cases/server/connectors/cases/cases_connector_executor.ts

```diff
@@ -630,11 +636,18 @@ export class CasesConnectorExecutor {
       return casesMap;
     }
 
+    const configurations = await this.casesClient.configure.get();
```

Comment:
Should I catch possible errors here and on line 893 @cnasikas ?

Reply:
> The connector's runner will catch any errors and handle them accordingly.

I also added a test in `cases_connector_executor.test.ts` for when `casesClient.configure.get` fails 👍